examples/spark_udf/spark_udf.py

```diff
@@ -1,9 +1,20 @@
 import mlflow
 from sklearn import datasets
 from sklearn.neighbors import KNeighborsClassifier
+from pyspark import SparkConf
 from pyspark.sql import SparkSession
 
-spark = SparkSession.builder.getOrCreate()
+
+conf = SparkConf()
+# conf.set(key="spark.python.worker.reuse", value="true")
+# conf.set(key="spark.task.maxFailures", value="1")
+# conf.set(key="spark.sql.execution.pyspark.udf.simplifiedTraceback.enabled", value="false")
+# conf.set(key="spark.sql.pyspark.jvmStacktrace.enabled", value="true")
+conf.set(key="spark.driver.memory", value="4g")
```

Comment:
Do the VMs that run on Github actions have enough free memory for this? That's a LOT of DRAM to allocate for such a small data set. Maybe try taking this down for a local test of this nature? 1g? 500M?